gems/smithy/lib/smithy/views/client/client.rb

```diff
@@ -12,16 +12,16 @@ def initialize(plan, code_generated_plugins)
           super()
         end
 
-        def namespace
-          @plan.gem_namespace
+        def module_name
```

Comment:
Every single view we have needs to do this - maybe we should move this to our [View](https://github.com/smithy-lang/smithy-ruby/blob/decaf/gems/smithy/lib/smithy/views/view.rb) class that all of these inherit?

Reply:
Good question. I'm not sure. Originally I had that in View but pulled it out. I don't know how View will impact servers and whether these methods would exist. We could stuff them in View, but also, it might be good for each view to be explicit about what methods they need.